changelog.d/pr-7358

```diff
@@ -0,0 +1,21 @@
+synopsis: Enable HPC coverage for `cabal-install`
```

Comment:
I think the main point _for users_ is the cabal-install-solver/cabal-install-library stuff more than hpc